src/diffusers/models/transformers/transformer_flux.py

```diff
@@ -161,6 +162,7 @@ def forward(
         encoder_hidden_states: torch.FloatTensor,
         temb: torch.FloatTensor,
         image_rotary_emb=None,
+        joint_attention_kwargs={},
```

Comment:
can you explain what additional argument you need to pass down to flux attention processor?

Reply:
Thank you for your recognition! 

In our work, I am trying to integrate box and mask into the FLUX model and implement layout control (similar to what has been done in many works on SD1.4). This requires modifying the attention processor. I believe that the architecture of FLUX and other transformers can also be used to develop better layout control algorithms, so I believe these modifications will contribute to future training-free experiments on FLUX.